### PR TITLE
Added missing translations for wipeing city and additional DE translations

### PR DESCRIPTION
**Pull request by Akurosia**

### CODE REVIEW

File: ui/raidboss/data/03-hw/alliance/weeping_city.js [hunk 1]
```diff
@@ -67,13 +68,23 @@
...
+      netRegexKo: NetRegexes.message({ line: '.*거미 여왕의 사냥터 will be sealed off.*?', capture: false }),
       run: function(data) {
         data.arachneStarted = true;
       },
     },
     {
       id: 'Weeping City HeadMarker Ozma',
       netRegex: NetRegexes.message({ line: '.*Gloriole will be sealed off.*?', capture: false }),
+      netRegexDe: NetRegexes.message({ line: '.*金字塔上层 will be sealed off.*?', capture: false }),
+      netRegexFr: NetRegexes.message({ line: '.*Aureole will be sealed off.*?', capture: false }),
```
Comment (quisquous) [inline ui/raidboss/data/03-hw/alliance/weeping_city.js:84]: I'm surprised this does not give extra errors given that the regex replacement is `The Gloriole`.  Would you mind removing `.*` from the beginning of each of these zone regexes and changing `Queen's Room` and `Gloriole` to `The Queen's Room` and `The Gloriole`? Sorry for not catching this the first time around.

**Comment by JLGarber** [comment]
Thanks for taking care of the translations!

File: ui/raidboss/data/03-hw/alliance/weeping_city.js (hunk 1 above)
Comment (Akurosia) [reply ui/raidboss/data/03-hw/alliance/weeping_city.js]: done

File: ui/raidboss/data/03-hw/alliance/weeping_city.js [hunk 2]
```diff
@@ -67,37 +67,37 @@
...
     },
     {
       id: 'Weeping City HeadMarker Calofisteri',
-      netRegex: NetRegexes.message({ line: '.*Tomb Of The Nullstone will be sealed off.*?', capture: false }),
-      netRegexDe: NetRegexes.message({ line: '.*Kammer des Nullsteins will be sealed off.*?', capture: false }),
-      netRegexFr: NetRegexes.message({ line: '.*Tombeau de la Clef de voûte will be sealed off.*?', capture: false }),
-      netRegexJa: NetRegexes.message({ line: '.*要の玄室 will be sealed off.*?', capture: false }),
-      netRegexCn: NetRegexes.message({ line: '.*契约石玄室 will be sealed off.*?', capture: false }),
-      netRegexKo: NetRegexes.message({ line: '.*쐐기 안치소 will be sealed off.*?', capture: false }),
+      netRegex: NetRegexes.message({ line: 'Tomb Of The Nullstone will be sealed off.*?', capture: false }),
```
Comment (quisquous) [inline ui/raidboss/data/03-hw/alliance/weeping_city.js:95]: This needs a "The" as well, I think?

Comment (quisquous) [reply ui/raidboss/data/03-hw/alliance/weeping_city.js:95]: Your translations need it as well.

`00|2020-07-18T10:24:01.0000000-07:00|0839||The Tomb of the Nullstone will be sealed off in 15 seconds!|`

Comment (Akurosia) [reply ui/raidboss/data/03-hw/alliance/weeping_city.js]: hopefully i did not missed any of it :)
getting late here xD

**Comment by quisquous** [comment]
Thanks for fixing these :)

**Pull request merged**